Update_CRO_integreatly-operator.md

```diff
@@ -5,7 +5,8 @@ add the latest version of CRO
 ## Release new version bundle of Cloud Resource Operator
 
 From your CRO branch, navigate to makefile and ensure that:
-- PREVIOUS_VERSION and VERSION match the desired versions.
+- PREVIOUS_VERSION and VERSION match the desired versions. You can confirm the PREVIOUS_VERSION by checking the last release CRO in rhmi_types
+version e.g. rhoam-release-v1.17.0 https://github.com/integr8ly/integreatly-operator/blob/rhoam-release-v1.17/apis/v1alpha1/rhmi_types.go#L96
```

Comment:
Do you think adding a note about "ensure that CRO has not been released in current sprint" makes sense?

Reply:
will add a note